.github/renovate.json5

```diff
@@ -18,7 +18,14 @@
       matchManagers: ["maven", "gradle", "gradle-wrapper"],
       matchUpdateTypes: ["major"],
       automerge: false
-    }
+    },
+
+    // Reduce awssdk update frequency (which has daily releases)
+    {
+      matchManagers: ["maven", "gradle"],
+      matchPackageNames: ["software.amazon.awssdk:*"],
+      extends: ["schedule:weekly"],
```

Comment:
Should we permanently disable it and manually update it only with Iceberg version bump? 
Iceberg may expect the matching version of aws sdk from the release (But latest version will work as long as the compatibility is not broken)

Reply:
i think the cross engine tests (which dont run on github) would tell us if such an upgrade broke compatibility... imo reducing by factor 7 should be good enough for now.